### PR TITLE
Implemented tests for book#index and book#show

### PR DESCRIPTION
**Pull request by marcegl91**
### Summary

- Implemented tests for book#index and book#show

### Trello Card

https://trello.com/c/KT3eY0kx/22-crear-tests-para-rutas-index-y-show-de-book

### CODE REVIEW

File: wbooks-api/spec/controllers/api/v1/books_controller_spec.rb [hunk 1]
```diff
@@ -1,4 +1,53 @@
...
+      before do
+        get :index
+      end
+
+      it 'responses with all the books' do
+        expected = JSON.parse(ActiveModel::Serializer::CollectionSerializer.new(
+          books, serializer: BookIndexSerializer
+        ).to_json)
+
+        content = JSON.parse(response.body)['page']
```
Comment (facucompe) [inline wbooks-api/spec/controllers/api/v1/books_controller_spec.rb]: we always define a helper to do this. Add this:
```
module Response
  module JSONParser
    def response_body
      ActiveSupport::JSON.decode(response.body) if response.present?
    end
  end
end
```
in `spec/support/parsed_response_helper.rb` . Also add it in `rails_helper`

**Review by facucompe — CHANGES_REQUESTED** [review]
Just that small change

**Review by facucompe — APPROVED** [review]

**Review by floresv — CHANGES_REQUESTED** [review]
and failure cases?

**Review by facucompe — APPROVED** [review]

**Review by floresv — APPROVED** [review]
LGTM

**Pull request merged**